cookin/app/templates/index.html

```diff
@@ -105,6 +105,19 @@
                             {% for tag in recipe.recipe_tags.all %}
                             <li>{{ tag }}</li>
                             {% endfor %}
+                            {% for t in tools%}
```

Comment:
This part should not be nested inside of "if recipe.recipe_tags.all" which checks if recipe has tags. It can be put in a separate if statement that checks if recipe has tools.